docs/reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters.md

```diff
@@ -76,6 +76,74 @@ With this endpoint enabled for the downstream cluster, Rancher generates an extr
 
 You will need to use a context defined in this kubeconfig file to access the cluster if Rancher goes down. Therefore, we recommend exporting the kubeconfig file so that if Rancher goes down, you can still use the credentials in the file to access your cluster. For more information, refer to the section on accessing your cluster with [kubectl and the kubeconfig file.](../../how-to-guides/new-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md)
 
+## Impersonation
+
+Users technically exist only on the upstream cluster. When users interact with a downstream cluster through the authentication proxy, there needs to be some entity downstream to serve as the actor for those requests. Rancher creates service accounts to be that entity. Each service account is only granted one permission, which is to **impersonate** the user they belong to. If there was only one service account that could impersonate any user, then it would be possible for a malicious user to corrupt that account and escalate their privileges by impersonating another user. This issue was the basis for a [CVE](https://github.com/rancher/rancher/security/advisories/GHSA-pvxj-25m6-7vqr).
```

Comment:
> If there was only one service account that could impersonate any user, then it would be possible for a malicious user to corrupt that account and escalate their privileges by impersonating another user. This issue was the basis for a [CVE](https://github.com/rancher/rancher/security/advisories/GHSA-pvxj-25m6-7vqr).

I feel like this part of the text isn't necessary. This paragraph seems more clear to me without it, not sure if you agree though. Do you feel the need to explain the design decision wrt. impersonation?

Reply:
I like this section just because it was my first question after reading the above statement.